test/mocks/intent.ts

```diff
@@ -3,8 +3,10 @@ export class MockIntent {
     public ensureRegisteredCalled: boolean = false;
     public leftRoom: string = "";
     public clientJoinRoomCalledWith: {roomString: string, opts: any}|null = null;
-    constructor(public userId: string) {
+    public userId: string;
 
+    constructor(userId: string) {
+        this.userId = userId;
```

Comment:
OOI why did this need to be changed?  Is/was there an issue with using empty constructor bodies or something?